nntrainer/tensor/tensor.cpp

```diff
@@ -170,7 +170,7 @@ Tensor Tensor::Map(float *buf, unsigned int size, const TensorDim &d,
       "[Tensor::Map] empty tensor dim is not allowed");
   }
 
-  if (d.getDataLen() + offset > size) {
+  if (d.getDataLen() * sizeof(float) + offset > size) {
```

Comment:
it would be better to rename size to bytes or size_bytes to avoid any confusion.

Reply:
Done